docs/examples/openebs.md

```diff
@@ -0,0 +1,161 @@
+# OpenEBS
+
+This tutorial covers the installation of OpenEBS as a helm chart, both from
+scratch and how to migrate it to a helm extension.
+
+## Installing OpenEBS from scratch
+
+**WARNING**: Do not configure OpenEBS both as a storage extension and helm
+extension. Because this is considered an invalid configuration k0s will ignore
+the configuration file entirely in order to prevent accidental upgrades or
+downgrades. The chart objects defined in the API will still have normal behavior.
+
+OpenEBS can be installed as a helm chart by adding it as an extension to your configuration:
+
+```yaml
+  extensions:
+    helm:
+      repositories:
+      - name: openebs-internal
+        url: https://openebs.github.io/charts
+      charts:
+      - name: openebs
+        chartname: openebs-internal/openebs
+        version: "3.9.0"
+        namespace: openebs
+        order: 1
+        values: |
+          localprovisioner:
+            hostpathClass:
+              enabled: true
+              isDefaultClass: false
+```
+
+If you want OpenEBS to be your default storage class set `isDefaultClass` to `true`.
+
+## Migrating bundled OpenEBS to helm extension
```

Comment:
Can we have an integration test that tests the migration? I'm not sure yet how the managed extension file will disappear from the helm folder of existing controllers, and how it will be prevented to be purged by the manifest applier. We should remove the stack annotation from the existing chart, too.

Reply:
I second to have an integration test for this.